authserver/app/app.py

```diff
@@ -41,11 +45,28 @@ def create_app(environment: str = None):
     @app.after_request
```

Comment:
The "print" function may not be the best way to output here. I recommend using the Python logging library with a loghandler for stdout. Print can lead to performance issues.

Reply:
Resolved using the logging library with custom format.